NitroxClient/GameLogic/Entities.cs

```diff
@@ -102,35 +101,29 @@ public void BroadcastEntitySpawnedByClient(WorldEntity entity)
         /// Also saves resources by using the IOut instances
         /// </remarks>
         /// <param name="forceRespawn">Should children be spawned even if already marked as spawned</param>
-        public IEnumerator SpawnBatchAsync(IEnumerable<Entity> batch, bool forceRespawn = false, bool skipFrames = true)
+        public IEnumerator SpawnBatchAsync(List<Entity> batch, bool forceRespawn = false, bool skipFrames = true)
```

Comment:
If we reimplement the Stack/LinkedList would change this back to IEnumerable and remove any `ToList()` cast from callers